java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java

```diff
@@ -131,10 +131,12 @@ public static String sanitizeString(String str) {
       index = sanitized.indexOf('-');
     }
 
-    sanitized = sanitized.replace('.', '_');
-    sanitized = sanitized.replace(' ', '_');
-    sanitized = sanitized.replace('\'', '_');
-    sanitized = sanitized.replace('\"', '_');
+    StringBuilder sanitizedSb = new StringBuilder(sanitized.length());
+    for (char originalChar : sanitized.toCharArray()) {
+      char sanitizedChar = Character.isJavaIdentifierPart(originalChar) ? originalChar : '_';
```

Comment:
[nitpick] I noticed only now looking at the API, but at index`[0]` we should probably use [isJavaIdentifierStart](https://docs.oracle.com/javase/8/docs/api/java/lang/Character.html#isJavaIdentifierStart-char-) .

Reply:
It would be possible to add an extra distinction here but I think it is not necessary since this check is already present for the first character: https://github.com/fabric8io/kubernetes-client/blob/master/java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java#L115-L119
If you want me to do it anyway, I can apply this suggestion: 
```suggestion
      boolean keepChar = sanitizedSb.length() == 0 ? Character.isJavaIdentifierStart(originalChar)
          : Character.isJavaIdentifierPart(originalChar);
      char sanitizedChar = keepChar ? originalChar : '_';
```

Reply:
you are right, thanks for noticing!